.github/workflows/versionci.yml

```diff
@@ -0,0 +1,20 @@
+name: Version CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.2.0", "3.2.2", "3.3.0"]
```

Comment:
Is there a way to specify "all patch versions greater than 3.2.0"? Or better yet, "all latest stable releases greater than X"? Or will we need to update this as and when new releases have come out?